tests/homepage.spec.ts

```diff
@@ -0,0 +1,18 @@
+import { test, expect } from '@playwright/test';
+
+test('has title', async ({ page }) => {
+  await page.goto('/');
+
+  // Expect a title "to contain" a substring.
+  await expect(page).toHaveTitle(/Webnative SDK Demo/)
```

Comment:
i've tried importing the `appName` from `app-info` to use here, but imports from the other files in the app don't seem to work here. I may actually rewrite this test to make it do something more meaningful and less prone to error if things(like the app name) change

Reply:
I ended up removing this test because the other test that looks for and clicks the `register` button technically accomplishes the goal of checking if the home page loaded